EDAM_dev.owl

```diff
@@ -20365,14 +20365,18 @@
     <!-- http://edamontology.org/data_2751 -->
 
     <owl:Class rdf:about="http://edamontology.org/data_2751">
-        <rdfs:subClassOf rdf:resource="http://edamontology.org/data_2091"/>
-        <rdfs:subClassOf rdf:resource="http://edamontology.org/data_2903"/>
+        <oldParent rdf:resource="http://edamontology.org/data_2091"/>
+        <oldParent rdf:resource="http://edamontology.org/data_2903"/>
+        <rdfs:subClassOf rdf:resource="http://www.w3.org/2002/07/owl#DeprecatedClass"/>
         <created_in>beta12orEarlier</created_in>
         <oboInOwl:hasDefinition>An identifier of a particular genome.</oboInOwl:hasDefinition>
         <oboInOwl:inSubset rdf:resource="http://purl.obolibrary.org/obo/edam#data"/>
         <oboInOwl:inSubset rdf:resource="http://purl.obolibrary.org/obo/edam#edam"/>
         <oboInOwl:inSubset rdf:resource="http://purl.obolibrary.org/obo/edam#identifiers"/>
+        <oboInOwl:inSubset rdf:resource="http://purl.obolibrary.org/obo/edam#obsolete"/>
+        <oboInOwl:replacedBy rdf:resource="http://edamontology.org/data_2749"/>
```

Comment:
Although this might be the best option, let's for the sake of being on the safe side give it another thought here, about what the best `replacedBy` would be.
(see also a related [comment at BioPortal](https://bioportal.bioontology.org/ontologies/EDAM/?p=classes&conceptid=http%3A%2F%2Fedamontology.org%2Fdata_2749#notes))

Reply:
Agreed with @hmenager and @LucieLamothe to merge Genome identifier, Genome accession, and Genome build identifier, as these would be used randomly by most users

Reply:
The data_2749 is in any case the best bet for a replacement, although we could have been more careful and instead of `replacedBy` use `consider` for it. However, only used once, namely in https://bio.tools/genomereviews, and the Genome identifier/accession/build identifier need a revision.

Merging those concepts will be the rudimentary option, we can consider whether we need some distinctions, but then rather more obvious ones than current (_e.g._ by species, version, location, specimen, ...). New issue #833.